sigal/gallery.py

```diff
@@ -440,6 +443,13 @@ def thumbnail(self):
         return None
 
     @property
+    def rand_thumbnail(self):
+        try :
+            return join(self.name, random.choice(self.medias).thumbnail)
+        except IndexError:
+            return self.thumbnail
```

Comment:
Ok for renaming.

This function returns thumbnail attribute so the `url_from_path` is correctly done.